docs/Account.md

```diff
@@ -368,11 +375,8 @@ is_valid: felt
 
 This is the only external entrypoint to interact with the Account contract. It:
 
-1. Takes the input and builds a `Call` for each iterated message. See [Multicall transactions](#multicall-transactions) for more information
-2. Validates the transaction signature matches the message (including the nonce)
-3. Increments the nonce
-4. Calls the target contract with the intended function selector and calldata parameters
-5. Forwards the contract call response data as return value
+1. Validates the transaction signature matches the message (including the nonce)
+2. Calls _unsafe_execute.
```

Comment:
`__execute__` doesn't call `_unsafe_execute` directly! That's a subcall implementation detail. This section is confusing the external preset function with the library API. We need to distinguish between preset and library APIs -- something we've mentioned needs improvement across for the docs. For the time being, let's at least document in here that each preset's `__execute__` will call its corresponding library counterpart.

Then on each library function API doc (`execute` and `eth_execute`), we document they call `_unsafe_execute`.